File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikUnitsAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.queries.units.StepikUnitsGetQuery;

/**
 * @author meanmail
 */
public class StepikUnitsAction extends StepikAbstractAction {
    public StepikUnitsAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public StepikUnitsGetQuery get() {
        return new StepikUnitsGetQuery(this);
    }
}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikUnitsAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.queries.units.StepikUnitsGetQuery

class StepikUnitsAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun get(): StepikUnitsGetQuery {
        return StepikUnitsGetQuery(this)
    }
}
